Polyfills/Window/CMakeLists.txt

```diff
@@ -3,16 +3,31 @@ set(SOURCES
     "Source/Window.h"
     "Source/Window.cpp")
 
+if(WIN32)
+    set(SOURCES ${SOURCES} "Source/Windows/WindowPlatform.h")
+endif()
+
 add_library(Window ${SOURCES})
 warnings_as_errors(Window)
 
 target_include_directories(Window
     PUBLIC "Include")
 
+if(WIN32)
+    target_include_directories(Window 
+        PRIVATE "Source/Windows")
+endif()
+
 target_link_to_dependencies(Window 
     PUBLIC napi
     PRIVATE base-n
     PRIVATE JsRuntime)
 
+if(WIN32)
+    target_link_to_dependencies(Window
+        PRIVATE "shlwapi.lib")
```

Comment:
>shlwapi [](start = 17, length = 7)

Are we testing any uwp flavors? It looks currently like this has only been validated against win32

Reply:
Not yet, I noticed that your UWP support is in now, so I should support that as well now.